bridges/ETTVBridge.php

```diff
@@ -0,0 +1,142 @@
+<?php
+class ETTVBridge extends BridgeAbstract {
+
+	const MAINTAINER = "GregThib";
+	const NAME = 'ETTV';
+	const URI = 'https://www.ettv.tv/';
+	const DESCRIPTION = 'Returns list of 20 latest torrents for a specific search.';
+	const CACHE_TIMEOUT = 14400; // 4 hours
+
+	const PARAMETERS = array( array(
+		'query' => array(
+			'name' => 'Keywords',
+			'required' => true
+		),
+		'cat' => array(
+			'type' => 'list',
+			'name' => 'Category',
+			'values' => array(
+				'(ALL TYPES)'				=>  '0',
```

Comment:
Please put only one space between the `=>` and the value, same for the other lines.